airflow/providers/apache/druid/CHANGELOG.rst

```diff
@@ -27,6 +27,19 @@
 Changelog
 ---------
 
+3.6.0
+.....
+
+Features
+~~~~~~~~
+
```

Comment:
```suggestion
```

Reply:
We should remove this one - it's up to release manager to decide a version. I think it shoud be 4.0.0 as the connection change is breaking.

Reply:
Yes.
In changelog just place the info at the top. I will pick it and arrange it during release process.

I think we can agree this is a breaking change so
@jaegwonseo can you please add  4.0.0 entry in provider yaml?

Reply:
@potiuk @eladkal 
i'm sorry for late replying
i'm gonna do that tonight